TESTING.md

```diff
@@ -14,13 +14,17 @@ Overview
 - [Misc](#misc)
 
 # General information
-OpenSearch Dashboards uses [Jest](https://jestjs.io/) for unit and integration tests, [Selenium](https://www.selenium.dev/) for functional tests, and [Cypress](https://www.cypress.io/) for backwards compatibility tests.
+OpenSearch Dashboards uses [Jest](https://jestjs.io/) for unit and integration tests, [Cypress](https://www.cypress.io/) for backwards compatibility tests and functional tests and [Selenium](https://www.selenium.dev/) for some legacy functional tests (Tests should no longer be written in Selenium).
```

Comment:
What's test coverage of cypress test compare to selenium test

Reply:
Right now most of the tests are covered using Selenium. The goal now that we have the Test Orchestrator is that we start moving these to cypress and deperecate their selenium counter part. The effort for this is quite large so that migration will need to happen incrementally while all new tests are written in cypress, thus aiding this transition.